src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -29,6 +29,7 @@ Users who need manual instrumentation should reference the "Datadog.Trace" packa
 
   <ItemGroup>
     <ProjectReference Include="..\Datadog.Trace.ClrProfiler.Managed.Core\Datadog.Trace.ClrProfiler.Managed.Core.csproj" />
+    <ProjectReference Include="..\Datadog.Trace.DuckTyping\Datadog.Trace.DuckTyping.csproj" />
```

Comment:
Can you also add the project to the Datadog.Trace.Minimal solution? Otherwise it'll prevent it from compiling

Reply:
Oh you're right, totally forgot that solution.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/976/commits/177ff9aac1c98523d035e8be27434bad0c17802c